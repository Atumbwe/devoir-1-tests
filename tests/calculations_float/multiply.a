56.534 34.43 mul