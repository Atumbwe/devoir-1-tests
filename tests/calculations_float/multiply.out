 ------------------
 <   1946.46562   >
 ------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||